velox/functions/sparksql/tests/DecimalArithmeticTest.cpp

```diff
@@ -522,5 +545,50 @@ TEST_F(DecimalArithmeticTest, decimalDivTest) {
       {makeConstant<int128_t>(
           DecimalUtil::kLongDecimalMax, 1, DECIMAL(38, 0))});
 }
+
+TEST_F(DecimalArithmeticTest, notAllowPrecisionLoss) {
+  setDecimalOperationsAllowPrecisionLoss(false);
```

Comment:
Yes, because we construct the class Function by this config https://github.com/facebookincubator/velox/pull/10383/files#diff-4baee9f82f9c347f753972415d345941d2c2a110b608d480b090650171da096bR754

Reply:
And this function name only exists in test code